count-fasta-seqs.sh

```diff
@@ -105,3 +105,7 @@ echo "$@"
 #
 # ADD YOUR CODE BELOW:
 
+
+
+#Comment from CTracy: Good Job - there is just one additional line of code in here that is causing it to print an extra line.
+#See if you can find that line and comment it out and then reply to my comment below so I know you received this!
```

Comment:
#I have commented out that line of code "echo "$@"", but still the tests failed.
#Kindly, guide me what's the problem now!

Reply:
Moreover, the code is producing the same output as expected, now. But tests are still not working.